tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/SendMessageIntegration.cs

```diff
@@ -50,16 +51,24 @@ public class SendMessageIntegration
             // for the InjectHeadersIntoMessage<TSendMessageRequest> call below
             var requestProxy = request.DuckCast<ISendMessageRequest>();
 
+            var queueName = AwsSqsCommon.GetQueueName(requestProxy.QueueUrl);
             var scope = AwsSqsCommon.CreateScope(Tracer.Instance, Operation, out var tags, spanKind: SpanKinds.Producer);
             if (tags is not null && requestProxy.QueueUrl is not null)
```

Comment:
I'm not sure if the queue URL can be null, but we're checking for null here and getting the queueName in this block. LMK if the null-check is unnecessary, but we could probably get the queueName only once. And I think this comment applies to multiple integration files

Reply:
Ah yes I missed this one, I meant to reuse the `queueName` variable. `GetQueueName` is null-safe, however, we don't want to write the tag if the value is null, hence the null check.